source/_patterns/components/secondary-navigation/secondary-nav.md

```diff
@@ -0,0 +1,30 @@
+---
+title: PatternFly Secondary Navigation
+---
+## Overview
+
+The secondary navigation can be used on it's own but it's usually used with primary horizontal nav or tabs.
+
+## Accessibility
+`role="tabpanel"` should be used for the div that wraps the tab contents.
+- `aria-selected="true"` should be added to the anchor that is active.
+- `aria-disabled="true"` should be added to the anchor that is disabled.
+
+| Attribute | Value | Usage |
+| -- | -- | -- |
+| `role` **Applied to:** `<nav>` | "navigation" |  **Outcome:** [Refer to this explanation for more details](http://a11yproject.com/posts/aria-landmark-roles/)   **Required:** Yes, until implicit mapping for `<nav>` is fully supported  |
+| `role` **Applied to:** `<ul>` | "tablist" |  **Outcome:** Identifies the `<ul>` list as a tablist to assistive device users   **Required:** Yes |
+| `role` **Applied to:** `<a>` | "tab" |  **Outcome:** Identifies the `<a>` link as a tab to assistive device users   **Required:** Yes |
+| `role` **Applied to:** `<div>` that wraps the tab contents | "tabpanel" |  **Outcome:** Identifies the tab contents to assistive device users   **Required:** Yes  **Note:** And example of this is on the basic template |
+| `aria-selected` **Applied to:** `a` | "true" |   **Outcome:** Communicates which page is selected to assistive device users   **Required:** Yes |
+| `aria-disabled` **Applied to:** `a` | "true" |   **Outcome:** Communicates which page is disabled to assistive device users   **Required:** Yes |
+
+## Usage
+
+| Class | Usage |
+| -- | -- |
+| `.pf-secondary-nav` **Applied to:** `ul` |  **Outcome:** Initiates a secondary underline style menu **Required:** Yes **Remarks:** Always use inside a `.nav` and `.nav-tabs` |
+| `.pf-secondary-nav__item` **Applied to:** `li` |  **Outcome:** This hook is here to future proof the component **Required:** No |
```

Comment:
Maybe say "This hook is here to provide a place to style the nav item itself." That it is "future proof" doesn't necessarily tell me the purpose.